### ISSUE TITLE
Error when concatenating a param and a string in a LIKE filter. v2.1.16

### ISSUE THREAD

**Issue by peterperron** [orientechnologies/orientdb#6049]
#### This fails in v2.1.16:
`Select * From ACNodeAuthentication where acNodeID like ? + '%'`

It used to work in Orient DB 2.0.3

#### It works without the param:
`Select * From ACNodeAuthentication where acNodeID like '89837db0' + '%'`

It also works if I pass the % in the param value instead of concatenating in the query and it also works if I use a js function to concat: `where acNodeID like concatStr(?, '%')`
#### The error:
```
Encountered " "+" "+ "" at line 1, column 58.
Was expecting one of:
    <EOF> 
    <AND> ...
    <OR> ...
    <ORDER> ...
    <GROUP> ...
...
```

**Comment by luigidellaquila** [comment]
Hi @peterperron 

it seems that params are not correctly accepted as arguments in expressions. I'll check it asap

Thanks

Luigi

**Comment by luigidellaquila** [comment]
Hi @peterperron 

I just pushed a fix on 2.1.x branch, now I'm porting it to develop

Thanks

Luigi

**Comment by luigidellaquila** [comment]
Ok, pushed to develop as well. 

I'm closing this issue, please reopen if you see any other problems

Thanks

Luigi
